### PR TITLE
adding US09_HamidAlakbarov

### PR DESCRIPTION
**Pull request by hamidalakbarov**
Please review my changes, thanks!

### CODE REVIEW

File: configuration.properties [hunk 1]
```diff
@@ -1,4 +1,5 @@
 browser=chrome
 url=https://qa.meetsky.net
 username=User1
-password=Userpass123
\ No newline at end of file
+password=Userpass123
```
Comment (Oleksandrozhoh) [inline configuration.properties:4]: Interesting that even if you removed and added same lines of code back - we can see the history...

Comment (hamidalakbarov) [reply configuration.properties:4]: Yeah, seeing history is super useful

**Review by DanSpicica — COMMENTED** [review]
As usual Hamid, great thinking!

**Review by Oleksandrozhoh — APPROVED** [review]
Do not see anything wrong with this PR, looks great, robot class logic looks a bit complicated, we need you to help us to understand it better... during the next lunch and learn session!

File: src/test/java/net/meetsky/utilities/BrowserUtils.java [hunk 2]
```diff
@@ -88,4 +88,9 @@ public static void waitForInvisibilityOf(WebElement target) {
         WebDriverWait wait = new WebDriverWait(getDriver(), Duration.ofSeconds(10));
         wait.until(ExpectedConditions.invisibilityOf(target));
     }
+
+    public static void waitForElementToBeClickable(WebElement element) {
+        WebDriverWait wait = new WebDriverWait(Driver.getDriver(),Duration.ofSeconds(10));
+        wait.until(ExpectedConditions.elementToBeClickable(element));
+    }
```
[the remark this answers is not in the document]
Comment (hamidalakbarov) [reply src/test/java/net/meetsky/utilities/BrowserUtils.java:95]: Thanks Alex for feedback, the sequence of steps a little different for Mac and Windows.

**Comment by hamidalakbarov** [comment]
> As usual Hamid, great thinking!

Don't forget to approve PR please

**Review by latifaforozan — APPROVED** [review]
Very very well DONE Hamid. cant wait to hear the useCase of interface here.

**Comment by hamidalakbarov** [comment]
> Very very well DONE Hamid. cant wait to hear the useCase of interface here.

Thank you Latifa, I will talk about it tomorrow

**Review by evabaldwin — COMMENTED** [review]
Great job Hamid, as always! I'm glad you figured out the robot class, since it is coming from Java I'm sure it's more advanced than Selenium Actions

**Comment by hamidalakbarov** [comment]
> Great job Hamid, as always! I'm glad you figured out the robot class, since it is coming from Java I'm sure it's more advanced than Selenium Actions

Hi Eva, thanks for feedback, yes it was a good experience, I will share it with you guys

**Pull request merged**